Python/Sub-Arrays/README.md

```diff
@@ -1 +1,16 @@
-# Algorithms on Sub-Arrays
\ No newline at end of file
+# Algorithms on Sub-Arrays
```

Comment:
I don't think you are supposed to change the readme of subarrays folder here.

Reply:
I had created a separate folder for Fractional Knapsack under Greedy Algorithm because I wanted to include some usage examples in README. However, I was asked to change it and instead update the common README. So I followed the same style here in advance.  
Still if changes in README are not allowed, I will revert it here as well as in Fractional Knapsack. Please let me know @zugzwang03 @Kumar-laxmi

Reply:
@stiwari-ds let it be don't revert now

Reply:
Alright. Will stick to this format. Thanks